src/newViews/AnalysisModule/Analysis/PillarList/index.tsx

```diff
@@ -0,0 +1,128 @@
+import React, { useEffect, useCallback, useState, useMemo } from 'react';
+import { _cs } from '@togglecorp/fujs';
+import {
+    Pager,
+    Container,
+    ListView,
+} from '@the-deep/deep-ui';
+
+import { useRequest, useLazyRequest } from '#utils/request';
+
+import {
+    MultiResponse,
+    AnalysisPillars,
+} from '#typings';
+
+import AnalysisPillar from '../AnalysisPillar';
+
+import styles from './styles.scss';
+
+const MAX_ITEMS_PER_PAGE = 5;
+const keySelector = (item: AnalysisPillars) => (item.id);
+
+interface Props {
+    className?: string;
+    createdAt: string;
+    modifiedAt: string;
+    activeProject: number;
+    onAnalysisPillarDelete: () => void;
+    analysisId: number;
+}
+
+function AnalysisDetails(props: Props) {
+    const {
+        className,
+        createdAt,
+        modifiedAt,
+        activeProject,
+        onAnalysisPillarDelete,
+        analysisId,
+    } = props;
+
+    const [activePage, setActivePage] = useState<number>(1);
+
+    const queryOptions = useMemo(() => ({
+        offset: (activePage - 1) * MAX_ITEMS_PER_PAGE,
+        limit: MAX_ITEMS_PER_PAGE,
+    }), [activePage]);
+
+    const {
+        pending: pillarPending,
+        response: pillarResponse,
+        retrigger: pillarGetTrigger,
+    } = useRequest<MultiResponse<AnalysisPillars>>(
+        {
+            url: `server://projects/${activeProject}/analysis/${analysisId}/pillars/`,
+            method: 'GET',
+            query: queryOptions,
+        },
+    );
+
+    // NOTE: Whenever the details of the analysis is changed, we refetch all the pillar
+    // analysis of that analysis
+    useEffect(() => {
+        pillarGetTrigger();
+    }, [pillarGetTrigger, modifiedAt]);
```

Comment:
Why not get call this when we are specifically modifying the "modifiedAt"

Reply:
The action that changes modifiedAt is at parent, that is why I've checked for change ni modifiedAt here.
Let me know if there is a better way to do this.

Reply:
Parent passess a function as prop and calls it.

Reply:
the action resides on child and not parent.

Reply:
oh no!